makefiles/libc/newlib.mk

```diff
@@ -67,7 +67,15 @@ ifeq (1,$(USE_NEWLIB_NANO))
   NEWLIB_NANO_INCLUDE_DIR ?= $(NEWLIB_INCLUDE_DIR)/newlib-nano
   # newlib-nano overrides newlib.h and its include dir should therefore go before
   # the regular system include dirs.
-  INCLUDES := -isystem $(NEWLIB_NANO_INCLUDE_DIR) $(INCLUDES)
+  ifeq (0,$(words $(wildcard $(NEWLIB_NANO_INCLUDE_DIR))))
```

Comment:
```make
ifeq (,$(wildcard $(NEWLIB_NANO_INCLUDE_DIR)))
```
should be enough, is it not?

Reply:
I agree.

I used the `0, $(words` without thinking because in this PR https://github.com/RIOT-OS/RIOT/pull/8475 the last change was not working for @aabadie with `wildcard` (it was working on my computer).